xlators/storage/posix/src/posix-aio.c

```diff
@@ -402,6 +402,151 @@ posix_aio_writev(call_frame_t *frame, xlator_t *this, fd_t *fd,
     return 0;
 }
 
+int
+posix_aio_fsync_complete(struct posix_aio_cb *paiocb, int res, int res2)
+{
+    call_frame_t *frame = NULL;
+    xlator_t *this = NULL;
+    struct iatt prebuf = {
+        0,
+    };
+    struct iatt postbuf = {
+        0,
+    };
+    int _fd = -1;
+    int op_ret = -1;
+    int op_errno = 0;
+    int ret = 0;
+    fd_t *fd = NULL;
+
+    if (!paiocb) {
```

Comment:
Can this be NULL? If we go to out from here,  STACK_UNWIND_STRICT will be passed a null frame.

Reply:
It shouldn't. Maybe better relaxed to GF_ASSERT().